ivy_tests/test_ivy/test_frontends/test_numpy/test_statistics/test_averages_and_variances.py

```diff
@@ -256,10 +256,9 @@ def test_numpy_cov(
     test_flags,
     fn_tree,
     on_device,
+    keep_dims,
 ):
-    input_dtypes, x, axis = dtype_and_x
-    if isinstance(axis, tuple):
-        axis = axis[0]
+    input_dtypes, x = dtype_and_x
 
     np_frontend_helpers.test_frontend_function(
         input_dtypes=input_dtypes,
```

Comment:
Are those changes intentional? if not we should revert them.

Reply:
While I was testing nanpercentile, the numpy_cov tests were failing because of some error related to axis (not sure if it is an unexpected parameter since the cov function doesn't take in axis)

Reply:
Well in that case, we should make a PR for fixing the other tests just to keep things clean and easy to track.